dbhydra/dbhydra_core.py

```diff
@@ -1538,7 +1538,12 @@ def insert(self, rows, batch=1, replace_apostrophes=True, try_mode=False, debug_
                 query = "INSERT INTO " + self.name + " ("
                 for i in range(start_index, len(self.columns)):
                     if i < len(rows[k]) + 1:
-                        query += self.columns[i] + ","
+                        # column name containing space needs to be wrapped in `...`, otherwise causes syntax error
+                        if " " in self.columns[i]:
+                            column_name = '`' + self.columns[i] + '`'
```

Comment:
can you change to normal apostophes? ' i.e. use "'"

Reply:
no

Reply:
isn't stripping of the column name better approach here?